runtime/common/src/routing.rs

```diff
@@ -0,0 +1,73 @@
+use cfg_traits::{
+	liquidity_pools::{MessageSender, RouterSupport},
+	PreConditions,
+};
+use cfg_types::domain_address::{Domain, DomainAddress};
+use frame_support::{
+	dispatch::DispatchResult,
+	pallet_prelude::{Decode, Encode, MaxEncodedLen, TypeInfo},
+};
+use pallet_axelar_router::AxelarId;
+use sp_core::{H160, H256};
+use sp_runtime::traits::{BlakeTwo256, Hash};
+use sp_std::{marker::PhantomData, vec, vec::Vec};
+
+/// Identification of the router where the message is sent and received
+/// RouterId is more specific than Domain, because RouterId also identify by
+/// where the message is sent/received
```

Comment:
Nit: Make it a bit clearer.
```suggestion
/// Identification of the router where the message is sent and received.
/// 
/// NOTE: RouterId is more specific than Domain, because RouterId also identifies
/// the source and destination of the message.
```

Reply:
Thanks for always polishing the docs

Reply:
Just a though, not sure if RouterId identifies the source/destination, it identifies the "channel" to reach that destination

Reply:
Not sure the exact word instead of channel, maybe the medium? or, "how the message is sent/received"

Reply:
In other words, to explain better, the `Domain` represents the source and destination, but `RouterId` also represents how to reach them

Reply:
I've rewrote over your changes the comment, tell me if now it makes more sense

Reply:
Perfect!